packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -16,8 +13,8 @@
 ## 1.26.0 - (July 20, 2023)
 
 * Added
+  * Added examples for `terra-worklist-data-grid` when using pinned columns.
```

Comment:
This should be under unreleased.

Reply:
Actually, this should be fixed by #1720 , so let's wait for that to get merged and see if there's any remaining changes to be made.

Reply:
@sdadn @trandrew1023 Merged

Reply:
Checked out the latest and it looks good to me, closing this PR. Thanks!